apps/web/scripts/installExternalCourses.py

```diff
@@ -0,0 +1,20 @@
+import json
+import sys
+import os
+
+def main():
+    if len(sys.argv) != 2:
```

Comment:
what do you think about using the[ click library](https://click.palletsprojects.com/en/8.0.x/) for this? I know this is a simple script, but click does not need boilerplate at all. It would be like this basically:

```python
import click

@click.command()
@click.argument('config_file')
def main(config_file):
    pass
```

it does the argument parsing for you and even generates the error messages and `--help`

Reply:
Maybe another time.